tests/test_hierarchy_helpers.hpp

```diff
@@ -102,9 +102,19 @@ class DiscontinuousMaterialProperty : public dealii::Function<dim>
 
   {
     double value = 10.;
-    for (unsigned int d = 0; d < dim; ++d)
-      if (p[d] > 0.5)
-        value *= value;
+    for (unsigned int i = 0; i < 100; i += 2)
```

Comment:
Where is this hardcoded value of 100 coming from?

Reply:
I create a checkerboard of 100 by 100.

Reply:
Oh, I think I see what you are doing. Wouldn't it be easier if instead of looping, you would just do something like this:
```c++
unsigned int dim_scale = 0;
for (unsigned int d = 0; d < dim; ++d)
  dim_scale += (long(floor(p[d]*100)) % 2);

return (dim_scale == dim ? 100. : 10.);
```

Reply:
sure